meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -10,7 +10,7 @@ PARTITION_SIZE=${PARTITION_SIZE:=1024}
 
 # Create a BTRFS disk image
 dd if=/dev/zero of=/export/data_disk.img bs=1M count=$PARTITION_SIZE
-mkfs.btrfs /export/data_disk.img
+mkfs.btrfs --mixed --metadata=single /export/data_disk.img
```

Comment:
Does `--metadata=single` increase the possibility of corruption? Have we investigated how the mixed mode performs at low disk space? Have we tested whwether the emergency reserve works correctly?